src/SDKs/DevSpaces/Management.DevSpaces/Microsoft.Azure.Management.DevSpaces.csproj

```diff
@@ -6,12 +6,12 @@
     <PropertyGroup>
 		<PackageId>Microsoft.Azure.Management.DevSpaces</PackageId>
 		<Description>Provides management operations support for Azure Dev Spaces.</Description>
-		<Version>0.11.0-preview</Version>
+		<Version>1.0.0</Version>
```

Comment:
@sanar-microsoft Just to confirm you wish to release a new version of the rest api with a stable Nuget package. We usually recommend Nuget versions to be preview for a first few versions before marking it as stable since we cannot accept breaking changes for a stable version

Reply:
I'll hold off on merging this then. I think we might want to change the validation rule for our resource name before releasing the stable version. I'll update this PR when the swagger changes are in. 

Thanks for the heads up.

Reply:
@sanar-microsoft you can still release this as a preview if you wish. Alternatively, please close this PR and open a new PR when you are ready to publish the stable version